distribution/lib/Standard/Base/0.0.0-dev/src/Data/Noise/Generator.enso

```diff
@@ -30,7 +30,7 @@ type Generator
        The return type may be chosen freely by the generator implementation, as
        it usually depends on the generator and its intended use.
     step : Number -> Interval -> Any
-    step _ _ = Unimplemented "Only intended to demonstrate an interface."
+    step _ _ = unimplemented "Only intended to demonstrate an interface."
```

Comment:
Wrong, it probably was working. `Main` does `from Error.Common export all`. This includes the `unimplemented` method, as long as you call it uppercased. Therefore it will work now in a much less surprising way.

Reply:
Yeap, that was long time coming